### PR TITLE
Prime Numbers

### PR DESCRIPTION
**Pull request by cpjiang**
Write a program to read a number N from the user and then find the first N primes. A prime number is a number that only had two divisors, one and itself.

### CODE REVIEW

File: MIT C++ Home Assignment/MIT C++ Home Assignment/main.cpp [hunk 1]
```diff
@@ -8,21 +8,51 @@
...
+            counter++;
+        }
+        next++;
+    }
+}
+
+int main() {
+    
+    cout << "Enter a number: " << flush;
+    int N;
```
Comment (chrl1638) [inline MIT C++ Home Assignment/MIT C++ Home Assignment/main.cpp]: People usually don't use capital case fro variable.

File: MIT C++ Home Assignment/MIT C++ Home Assignment/main.cpp [hunk 2]
```diff
@@ -8,21 +8,51 @@
 
 #include <iostream>
 using namespace std;
+bool isaprime(int n) {
```
Comment (cpjiang) [inline MIT C++ Home Assignment/MIT C++ Home Assignment/main.cpp]: 😀 Yeah! it works!

File: MIT C++ Home Assignment/MIT C++ Home Assignment/main.cpp [hunk 3]
```diff
@@ -8,21 +8,51 @@
```
Comment (chrl1638) [inline MIT C++ Home Assignment/MIT C++ Home Assignment/main.cpp]: U need to add description of the project here

**Comment by chrl1638** [comment]
LGTM

**Pull request merged**